heat/core/statistics.py

```diff
@@ -1103,9 +1185,54 @@ def mpi_argmin(a, b, _):
 MPI_ARGMIN = MPI.Op.Create(mpi_argmin, commute=True)
 
 
+def skew(x, axis=None, unbiased=True):
```

Comment:
Can we have parameter and return type annotations here (except DNDarray due to aliasing)